CHAINMAIL.md

```diff
@@ -0,0 +1,36 @@
+Part of the Agoric strategy for providing security is based on the
+[Formal Methods](https://agoric.com/papers/#formal-reasoning) literature, which has developed tools
+for proving that programs follow their specifications. There have been interesting advances in this
+area including the development of [seL4](https://sel4.systems/Info/FAQ/proof.pml) and the
+[proof of its correctness](https://sel4.systems/Info/FAQ/proof.pml). We have been collaborating with
+a research group at Victoria University who are investigating how to apply these techniques to
+secure distributed programs.
+
+The work with Victoria University
+[Reasoning about Risk and Trust in an Open World](https://ai.google/research/pubs/pub44272) has
+resulted in development of a specification language, Chainmail, which is intended to allow programs
+to be described with enough formality to investigate their security properties. We have used a
```

Comment:
Still sounds like we're claiming more than our current chainmail provides. Make explicit that it is only a subset that corresponds to a conventional object IDL, and that it starts with a syntactic subset of the Cap'n Proto IDL.

Reply:
Done.